addons/event_sale/models/event_registration.py

```diff
@@ -48,6 +49,26 @@ def create(self, vals):
                 subtype_id=self.env.ref('mail.mt_note').id)
         return res
 
+    def write(self, values):
+        if 'event_ticket_id' in values:
+            new_ticket_id = self.env['event.event.ticket'].browse(values['event_ticket_id'])
+            if self.event_ticket_id != new_ticket_id:
+                self._sale_order_ticket_type_change_notify(self, new_ticket_id)
+
+        result = super(EventRegistration, self).write(values)
+        return result
+
+    def _sale_order_ticket_type_change_notify(self, event_registration, new_ticket_id):
+        render_context = {
+            'registration': event_registration,
+            'old_ticket': event_registration.event_ticket_id.name,
+            'new_ticket': new_ticket_id.name
+        }
+        event_registration.sale_order_id.activity_schedule_with_view('mail.mail_activity_data_warning',
+             user_id=event_registration.event_id.user_id.id or self.env.ref("base.user_root").id,
```

Comment:
User root is not a real user. I think you could check for
1/ event user_id;
2/ order user_id;
3/ current user if not public;
4/ admin;